File: app/src/main/java/com/infosy/assingment/mycountry/activities/MainActivity.java
package com.infosy.assingment.mycountry.activities;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.infosy.assingment.mycountry.R;
import com.infosy.assingment.mycountry.ui.CountryDetailsFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, CountryDetailsFragment.newInstance())
                    .commitNow();
        }
    }
}

File: app/src/main/java/com/infosy/assingment/mycountry/activities/MainActivity.kt
package com.infosy.assingment.mycountry.activities

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.infosy.assingment.mycountry.R
import com.infosy.assingment.mycountry.ui.CountryDetailsFragment

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.container, CountryDetailsFragment.newInstance())
                    .commitNow()
        }
    }
}